### ISSUE TITLE
Null models

### ISSUE THREAD

**Issue by annaspiers** [eco4cast/neon4cast-beetles#6]
Follow-ups from July 21 meeting
- [ ] Performance metrics @Carl
- [ ] Add error to random walk and average baseline null forecasts for richness @taddallas 
- [ ] Build random walk and average baseline null forecasts for count @annaspiers 
- [ ] Pool subspecies to species @annaspiers 
- [ ] Data correction ( e.g., for trapnight, something about sampling effort) @brettmelbourne

**Comment by annaspiers** [comment]
There are 22 subspecies that share a species ID with another subspecies. That is, 22 subspecies would be pooled to 11 species. These 22 subspecies identify 3203 individuals. Should we consider leaving subspecies as subspecies?

Code to visualize this:
https://github.com/annaspiers/NEON-community-forecast/blob/d604716290dcc12b70ea76f7d6b41d4483b91789/00_clean-download-EDA.Rmd#L223-L250

**Comment by cboettig** [comment]
@annaspiers good question!  I could see the argument for going with a definition of richness as 'the finest resolution available' -- i.e. identifiably morphologically distinct species are maybe more likely to be functionally distinct and the 'species' level is somewhat arbitrary perhaps, but it does seem to open a can of worms ...  e.g. what about species that can be broken into subspecies by the para-taxonomist or expert taxonomist?  I think we should stick with lumping these.  From what you say, it sounds like `TaxonID` codes are specific to the species level, i.e. our metric of richness is essentially the number of unique TaxonIDs then.  Does that sound right?

**Comment by annaspiers** [comment]
@cboettig Right, I was hinting towards defining richness as 'the finest resolution available,' as you describe. 

Lumping subspecies is intuitive, although I don't follow the can of worms example - did you mean, "... that cannot be..."? Without looking into the spatial distribution of these 22 subspecies, I bet that pooling them into only 11 species IDs wouldn't make a qualitative difference in forecast results. This would be a good thing to test in our null models though.

The `taxonID` is unique to the parataxonomist's `scientificName` assignment. That is, different subspecies with the same pooled species ID have different `taxonID`s, but different morphospecies with the same species ID have the same `taxonID`. Thus, our metric of richness where we pool subspecies to the species level and then use unique `scientificName`-`morphospeciesID` IDs is not the same as unique `taxonID`s.
